src/app/commands/cmd_save_file.cpp

```diff
@@ -168,6 +170,14 @@ std::string SaveFileBaseCommand::saveAsDialog(
     saveDocumentInBackground(
       context, document,
       filename, markAsSaved);
+    Preferences::instance().document(document).saveCopy.filename("");
```

Comment:
Here we could use a temporal `auto& docPref = Preferences::instance().document(document)` variable instead of calling `Preferences::instance().document(document)` several times.

Other issue: I guess that we could use something like `docPref.saveCopy.aniDir(docPref.saveCopy.aniDir.defaultValue())`, etc. to reset each value.